packages/babel-parser/src/plugins/typescript.js

```diff
@@ -220,6 +220,25 @@ export default (superClass: Class<Parser>): Class<Parser> =>
       return result;
     }
 
+    tsParseImportType(): N.TsImportType {
+      const node: N.TsImportType = this.startNode();
+      this.expect(tt._import);
+      this.expect(tt.parenL);
+      if (!this.match(tt.string)) {
+        throw this.unexpected();
```

Comment:
Can we add a nice error message here please. Something like "A TypeScript type import requires a string literal as its argument."

Reply:
typescript reports here: `String literal expected.`

Reply:
but i'm ok with adding there whatever :)

Reply:
but we should be consistent about this, and in `tsParseExternalModuleReference` there is no message.

```
      if (!this.match(tt.string)) {
        throw this.unexpected();
      }
```

Reply:
@danez what do you think about `Argument in a type import must be an String Literal`

Reply:
👍